WTWSMS/decisions/realm_decisions.txt

```diff
@@ -1343,6 +1343,18 @@ decisions = {
 				factor = 0
 				religion = marcionism
 			}
+			modifier = {
+				factor = 10
+				trait = kind
+			}
+			modifier = {
+				factor = 100
+				trait = sympathy_judaism
+			}
```

Comment:
It's  called SYMPATHY FOR JUDAISM for a reason. They understand the suffering of Jews and will ALWAYS welcome them back.